### DISCUSSION TITLE
Release 7.2.1.0

### DISCUSSION THREAD

**Discussion by mikeatworkgp** [Courseplay/Courseplay_FS22#2447] [General]
Hallo es mir gestern aufgefallen das wenn ich zuerst den abfahrer  einstellen und dann den Rübenenter wird dieser von den CP abfahrer blockiert.  Das ganze Spiel ist blockiert. Wenn ich aber zuerst den Rübenernter einstellen und dann den abfahrer funktioniert es perfekt. 

Danke

**Comment by Tensuko** [comment]
Kann ich nicht reporduzieren, wenn du sowas meldest, dann bitte unter Issues mit ausgefülltenm bug template.
